settings.talon

```diff
@@ -1,4 +1,4 @@
-settings():
+:
```

Comment:
You have removed `settings()` from the start of the settings block. Can you put it back?

Reply:
oops, thanks and sorry